net_http_bsd/src/main.c

```diff
@@ -49,38 +55,67 @@ unsigned char readUTF(unsigned char c, int fd){
 	case 9516:return 0x12;
 	case 9524:return 0x11;
 	case 9532:return 0x15;
-	case 9600:return 0xDF;
-	case 9604:return 0xDC;
+	case 9600:return 0xC0;
+	case 9601:return 0xC0;
+	case 9602:return 0xDC;
+	case 9603:return 0xDC;
+	case 9604:return 0xC2;
+	case 9605:return 0xC2;
+	case 9606:return 0xDB;
+	case 9607:return 0xDB;
+	case 9608:return 0xDB;
 	}
 	return '?';
 }
 
 int main (int argc, char *argv[]){
-	static char net_mem[1*1024*1024];
+	(void)argc;(void)argv;
+	char* *url=NULL, *wttr[]={"wttr.in","/Paris"}, *rate[]={"rate.sx","/"};
```

Comment:
Please don't compact your variables declarations so much, it makes reading them harder and it doesn't play well with diff tools.